libobs/obs-module.c

```diff
@@ -357,9 +357,17 @@ static bool parse_binary_from_directory(struct dstr *parsed_bin_path,
 
 	dstr_copy_dstr(parsed_bin_path, &directory);
 	dstr_cat(parsed_bin_path, file);
+#ifdef __APPLE__
+	if (!os_file_exists(parsed_bin_path->array)) {
+		dstr_cat(parsed_bin_path, ".so");
```

Comment:
Is the duplication on line 370 intentional?

Reply:
This file is not modified in this PR. It may have been modified as part of one of its dependent PRs that has since been merged.